ocamltest/selftests/bug.c

```diff
@@ -0,0 +1,17 @@
+/*
+ Show that under Linux the core file is dumped in the directory
+ where the process is at crash time, rather than in its initial
+ current directory.
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+int main()
+{
+  char *p = NULL;
+  if ( chdir("/tmp") < 0) perror("chdir");
+  *p = 'a';
```

Comment:
This is not guaranteed to dump a core... You should use the `abort` function from the C standard library.